editor/reducer.js

```diff
@@ -441,13 +441,21 @@ export function mode( state = 'visual', action ) {
 	return state;
 }
 
-export function preferences( state = { isSidebarOpened: ! isMobile }, action ) {
+export function preferences( state = { isSidebarOpened: ! isMobile, panels: { 'post-status': true } }, action ) {
```

Comment:
This line is pretty long and we might consider breaking out the default to its own constant variable.

Reply:
Will fix the notes in #2568